src/app/report/edit-display/edit-display.component.html

```diff
@@ -38,7 +38,7 @@
   </div>
 </div>
 
-<p id="showRerunResult">
+<p id="showRerunResult" [style]="rerunBackgroundColor">
   {{ rerunResult }}
```

Comment:
However i would suggest leaving the color in the html and just using a boolean to determine which color to display:
```suggestion
<p id="showRerunResult" [ngStyle]="{'background-color': rerunSuccess ? 'green' : 'red'}">
  {{ rerunSuccess ? '[Rerun succeeded]' : '[Rerun failed]' }}
```